src/lib/components/NewLayeredMap/webgl/commands/drawWithAdvancedHillShading.js

```diff
@@ -300,7 +299,9 @@ function calcN(width, height) {
     }
     else if(resolution <= 810000) { // 900x900
         return 18;
+    } else if(resolution < 2293760) { // 1792x1280
+        return 8;
     }
 
-    return 8; // resoltion > 900x900
+    return 1; // resoltion > 900x900
```

Comment:
Doesn't this make it so that the default return is when resolution > 1792x1280?

Reply:
Yup, correct x)